src/macports1.0/macports.tcl

```diff
@@ -3644,6 +3647,44 @@ proc macports::_target_needs_deps {target} {
     }
 }
 
+##
+# Determine if the given target needs a toolchain of the given port needs
+# a toolchain. Returns true iff the target will require a compiler (or
```

Comment:
Editing error: the first "needs a toolchain" should be deleted.

Reply:
Done.